sources/editor/Stride.GameStudio.Avalonia/ViewModels/MainViewModel.cs

```diff
@@ -136,6 +141,21 @@ private Task OnOpen(UFile? initialPath)
         return OpenSession(initialPath);
     }
 
+    [RelayCommand]
```

Comment:
Use ICommandBase which already implements the command pattern with additional features.

Reply:
Yes, but I though that It might be better to reduce redundant code by using code generators like community toolkit have

Reply:
Anyway I've applied the changes

Reply:
Once we have finished the rewrite, maybe we can look at it. But for now we have code that is shared or duplicated between WPF and Avalonia, and I prefer to not have two diferent implementations at the same time.

Reply:
Right that's not urgent, so we can discuss this later